tool/tctl/common/app_command.go

```diff
@@ -144,6 +144,14 @@ Fill out and run this command on a node to make the application available:
 
 Your application will be available at {{.app_public_addr}}.
 
+To run a Teleport demo MCP server for MCP access:
+
+> teleport app start \
+   --token={{.token}} \{{range .ca_pins}}
+   --ca-pin={{.}} \{{end}}
+   --auth-server={{.auth_server}} \
+   --mcp-demo-server
```

Comment:
Nit: I would probably not add this here tbh. I think this may confuse people who just want to start a regular app service, MCP seems to be a much more niche use-case for now.

Reply:
dropped.

[52507dc](https://github.com/gravitational/teleport/pull/56637/commits/52507dc9d6b21f9a4451513b5408cfd1ef0e90bd)

(add added the flag to `teleport configure`, which should make the getting started guide super easy)